server/src/main/java/com/coincoinche/events/TurnStartedEvent.java

```diff
@@ -0,0 +1,14 @@
+package com.coincoinche.events;
+
+public class TurnStartedEvent extends Event {
+  private String[] authorizedPlays;
```

Comment:
Can we change `authorizedPlays` to `legalMoves`? Since that's what we've been using.

Reply:
Indeed, better to keep consistent naming, I renamed all the occurrences.